src/storm-pars/modelchecker/instantiation/SparseInstantiationModelChecker.cpp

```diff
@@ -31,6 +31,11 @@ namespace storm {
         bool SparseInstantiationModelChecker<SparseModelType, ConstantType>::getInstantiationsAreGraphPreserving() const {
             return instantiationsAreGraphPreserving;
         }
+
+        template <typename SparseModelType, typename ConstantType>
+        bool SparseInstantiationModelChecker<SparseModelType, ConstantType>::getOriginalModel() const {
```

Comment:
Should return a SparseModelType

Reply:
I should not commit old versions. Thanks for spotting